### PR TITLE
mark XamarinLicense task as deprecated

### PR DESCRIPTION
**Pull request by madhurig**

### CODE REVIEW

**Comment by madhurig** [comment]
xlicense

File: Tasks/XamarinLicense/Strings/resources.resjson/en-US/resources.resjson [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "loc.friendlyName": "Xamarin License",
   "loc.helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?LinkID=613739)",
-  "loc.description": "Activate or deactivate Xamarin licenses",
+  "loc.description": "Activate or deactivate Xamarin licenses (deprecated; upgrade to free version of Xamarin: https://store.xamarin.com)",
```
Comment (jpricketMSFT) [inline Tasks/XamarinLicense/Strings/resources.resjson/en-US/resources.resjson]: Should there be Strings.json files as well in this checkin?

**Comment by jpricketMSFT** [comment]
one small comment - changes look good!

**Comment by madhurig** [comment]
Yes ENU strings are checked in

**Pull request merged**